smart-contracts/contracts/cl-vault/src/vault/range/modify_percentage.rs

```diff
@@ -0,0 +1,164 @@
+use cosmwasm_std::{
+    to_json_binary, Coin, Decimal256, DepsMut, Env, Response, SubMsg, SubMsgResult, Uint128,
+};
+use osmosis_std::types::osmosis::concentratedliquidity::v1beta1::MsgCreatePositionResponse;
+
+use crate::{
+    helpers::{get_one_or_two, get_unused_balances},
+    msg::{ExecuteMsg, MergePositionMsg},
+    reply::Replies,
+    rewards::CoinList,
+    state::{Position, CURRENT_POSITION_ID, POOL_CONFIG, POSITIONS},
+    vault::concentrated_liquidity::{create_position, get_position, withdraw_from_position},
+    ContractError,
+};
+
+/// increase the ratio that a position has in the vault for depositors
```

Comment:
What is the flow for add ratio and lower ratio, these two entry points are only updating the ratio value in the POSITION store. When it is supposed to reflect in the pool?

Reply:
Yes, it is only supposed to be reflected in user the ratio users deposit it, we want to give the strategists the freedom to separate the wanted incoming funds of the vault vs the actual funds in the pool